frontend/public/components/cluster-settings/cluster-settings-utils.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-undef */
+import * as React from 'react';
+import { Dropdown } from '../utils';
+import * as classNames from 'classnames';
+
+export const MappingMethod: React.FC<MappingMethodProps> = ({selectedMappingMethod, onChange, required=true}) => {
+  const mappingMethodChanged = (mappingMethod: string) => {
+    onChange(mappingMethod);
+  };
+  const mappingMethods: object = {
+    'claim': 'Claim',
+    'lookup': 'Lookup',
+    'add': 'Add',
+  };
+  return (
+    <div className="form-group">
+      <label className={classNames('control-label', {'co-required': required})} htmlFor="tag">Mapping Method</label>
+      <Dropdown dropDownClassName="dropdown--full-width" items={mappingMethods} selectedKey={selectedMappingMethod} title={mappingMethods[selectedMappingMethod]} onChange={mappingMethodChanged} />
+      <div className="help-block" id="mapping-method-description">
+        { /* TODO: Add doc link when available in 4.0 docs. */ }
+        Specifies how new identities are mapped to users when they log in.
+      </div>
+    </div>
+  );
+};
+export type MappingMethodProps = {
+  selectedMappingMethod: string;
```

Comment:
I'd call this `value`

```suggestion
  value: 'claim' | 'lookup' | 'add';
```